.gitignore

```diff
@@ -1,6 +1,7 @@
 .cache
```

Comment:
is .cache needed?

Reply:
I left it in to not create untracked files for other developers that already have a `.cache` folder.